### ISSUE TITLE
Optionally show metadata for dependencies

### ISSUE THREAD

**Issue by ajroetker** [sfackler/cargo-tree#3]
I'm not sure if this goes outside the scope of this plugin but I thought I'd raise an issue here to discuss the possibility of add ManifestMetadata information to the output of `cargo tree`.

For example, I have a small example patch which displays `license` and `repository` information for the dependencies:

```
[~/cargo-tree]$ ./target/debug/cargo-tree tree -p petgraph
petgraph v0.2.5 MIT/Apache-2.0 https://github.com/bluss/petulant-avenger-graphlibrary
└── fixedbitset v0.1.0 MIT/Apache-2.0 https://github.com/bluss/fixedbitset
```

Would this be something that would be reasonable to add to the `--verbose` output (maybe with another flag)?

**Comment by sfackler** [comment]
This seems like a thing that could be configured by a formatting flag. I've seen a couple of other command line utilities that support this kind of thing, but unfortunately can't remember their names right now. Something like:

```
$ cargo tree -p petgraph -f "{n} {v} {l} {r}"
```

Where `{n}` prints the dependency name, `{v}` prints the version, `{l}` prints the license and `{r}` prints the
repository.

**Comment by Nemo157** [comment]
A template parameter to print the list of active features (and maybe inactive with a minus in front or something) could be useful. Also maybe one to show the dependency version spec as well as the actual version.

**Comment by sfackler** [comment]
Fixed by #3
